src/SuperDumpService/Views/Home/_Threads.cshtml

```diff
@@ -176,7 +176,12 @@
 												<td class="nobreak stackmethodname">
 													<span class="@tagclasses">@(frame.ModuleName)!<b>@(frame.MethodName)</b>+@(frame.OffsetInMethod.ToString("x"))</span>
 													@if (frame.SourceInfo != null) {
-														<span class="sourceinfo">@frame.SourceInfo.FileName():@frame.SourceInfo.Line</span>
+														@if (frame.SourceInfo.IsLinkAvailable() && Model.SvnUrl != null && Model.SvnUrl != "") {
+															<span class="sourceinfo"><a _target="blank" href=@Model.SvnUrl@frame.SourceInfo.GetLink()>@frame.SourceInfo.FileName()</a>:@frame.SourceInfo.Line</span>
+														} else {
+															<span class="sourceinfo">@frame.SourceInfo.FileName():@frame.SourceInfo.Line</span>
```

Comment:
if you're interested, you could also create a TagHelper for this construct (see https://github.com/Dynatrace/superdump/blob/master/src/SuperDumpService/TagHelpers/PossibleLinkTagHelper.cs)